pom.xml

```diff
@@ -577,7 +577,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.32.Fork1</version>
+        <version>1.1.32.Fork2-SNAPSHOT</version>
```

Comment:
@jpinner we could... but why we need until we release a new netty version ? That said if you really need one I can cut a release. Just let me know buddy